Android/ConstraintLayout/samples/NuGet.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <packageSources>
+        <add key="BuildOutput" value="../output" />
```

Comment:
Might want to do the same here: https://github.com/xamarin/XamarinComponents/pull/682/files#r330617947